src/tests/uat/reputation_override_uat.py

```diff
@@ -132,19 +132,20 @@ def main():
 
     ReputationOverride.bulk_delete(cb, [it_tool_override.id, cert_override.id])
 
-    delete_query = cb.select(ReputationOverride) \
-                     .set_override_list("WHITE_LIST") \
-                     .set_override_type("IT_TOOL") \
-                     .sort_by("create_time", "asc")
-
-    assert len(delete_query) == 0
+    isDeleted = True
+    try:
+        cb.select(ReputationOverride, it_tool_override.id)
+        isDeleted = False
+    except ObjectNotFoundError:
+        pass
```

Comment:
Instead of using a boolean I would recommend importing pytest and using the following code as this will fail if the code does not raise the expected exception
```
with pytest.raises(ObjectNotFoundError):
        cb.select(ReputationOverride, it_tool_override.id)
```

Reply:
Done